### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by langeuh**
Please keep a changelog to describe the basics that changed
it's hard to follow through the commits.
It doesn't have to be detailed, but at least the most important changes/additions/fixes

### CODE REVIEW

**Comment by langeuh** [comment]
Just saw another issue about the changelog with the link to the documentation.
I didn't know it existed, so a quick shortcut would be handy.
Most people search for the CHANGELOG.md file 

Some really nice features have been added! keep up the work

**Comment by z-song** [comment]
Thanks！
